modules/users/config.ts

```diff
@@ -0,0 +1,11 @@
+export interface Config {
+    maxProfilePictureBytes: number;
+    allowedMimes?: string[];
+}
+
+export const DEFAULT_MIME_TYPES = [
+    "image/jpeg",
+    "image/png",
+    "image/gif",
```

Comment:
let's keep it to jpeg and png for now. we def should support webp, but they can be animated which i don't want to support yet.